andela/ah-fortem-frontend
# 166980309 Rate Articles

#### Title
Rate articles and display average

#### Description
Displays average rating and total rating count for all articles  and allows an authenticated user to rate or update his/her rating to an article

#### Type of change




### How can this be manually tested?

- Clone the repository  
  `git clone https://github.com/andela/ah-fortem-frontend.gi`

-  Check out to current branch
    `git checkout story/166980309-rate-articles`

- setup npm
`$ npm install`

- run application
`$ npm run dev`

- View Articles average star rating and total rating count

![image](https://user-images.githubusercontent.com/11487123/61505649-3fd5e500-a9e8-11e9-9b1d-1c0adf3e91bc.png)

- View average star rating for a single article

![image](https://user-images.githubusercontent.com/11487123/61505915-26816880-a9e9-11e9-82e5-5555d35b28ab.png)

- Log in to allow rating an article

![image](https://user-images.githubusercontent.com/11487123/61505793-ab1fb700-a9e8-11e9-81db-f28fc7dd10b8.png)

- Rate an article once you've logged in successfully

![image](https://user-images.githubusercontent.com/11487123/61505870-f639ca00-a9e8-11e9-8bad-18c41b989591.png)

![image](https://user-images.githubusercontent.com/11487123/61505839-dacebf00-a9e8-11e9-9084-4b6fb7dbb88d.png)


#### How Has This Been Tested?

#### Checklist:
- [x] Display average rating and rating count for all articles on the homepage
- [x] Add add rating functionality on an article

#### PT stories
[#166980309](https://www.pivotaltracker.com/story/show/166980309)
